internal/grpcrand/grpcrand.go

```diff
@@ -1,3 +1,6 @@
+//go:build !go1.21
+// +build !go1.21
```

Comment:
We don't need both syntaxes for this, since we don't support Go 1.16 or earlier.  Just `//go:build` is fine.

Reply:
Done